docs/CAD/fusion360course/CAD1-beginner.md

```diff
@@ -0,0 +1,257 @@
+# CAD-1 - Fusion Kurs Beginner
+
+- [CAD-1 - Fusion Kurs Beginner](#cad-1---fusion-kurs-beginner)
+  - [- Session 08: Flächen](#--session-08-flächen)
+  - [Vorbereitung](#vorbereitung)
+  - [Session 1: Einstieg](#session-1-einstieg)
+  - [Session 02: Deep Dive “Create”](#session-02-deep-dive-create)
+  - [Session3: Patterns!](#session3-patterns)
+  - [Session 04: Create Form (Sculpting Mode)](#session-04-create-form-sculpting-mode)
+  - [Session 05: Baugruppen](#session-05-baugruppen)
+  - [Session 06: Externe Dateien und Rendern](#session-06-externe-dateien-und-rendern)
+    - [Übersicht 3D-Dateiformate](#übersicht-3d-dateiformate)
+    - [Tutorial  und Dateien:](#tutorial--und-dateien)
+  - [Session 07: Sheet Metal](#session-07-sheet-metal)
+  - [Session 08: Flächen](#session-08-flächen)
+
```

Comment:
I think a TOC is necessary because there is so much info on the page. I will add the T[OC-plugin](https://github.com/mrpotatoes/docsify-toc) if thats okay

Reply:
I cant figure out how to make the TOC only for specific pages. It adds a TOC to every page. Is it possible to specify the pages ? if not-is a TOC on every page okay?